src/actions/index.js

```diff
@@ -15,6 +15,7 @@ import type {
 //
 // Action Types
 //
+export const IS_ONLINE_CHECK = 'IS_ONLINE_CHECK';
```

Comment:
I think a name like `CHECK_ONLINE_STATUS` would match the names of the other actions. 
It's always `<action-verb>-<what-is-it-doing>` e.g. `ADD_PROJECT`.
I'm not sure if `check` is the right word but it's probably OK. It's more an updating of the online status - so maybe a `SET_ONLINE_STATUS`. What do you think?

It would be also good to move that line to the other app-status constants at line 85. You can also change the comment in line 85 so it's clear that these are the actions used in `app-status`.

Reply:
I think you're right. `SET_ONLINE_STATUS` would be better suited as you're setting the app's online status.